web_widget_digitized_signature/static/src/js/digital_sign.js

```diff
@@ -22,8 +22,8 @@ odoo.define('web_widget_digitized_signature.web_digital_sign', function(require)
         placeholder: "/web/static/src/img/placeholder.png",
         init: function() {
             this._super.apply(this, arguments);
-            this.$('> img').remove();
-            this.$('.signature > canvas').remove();
+//            this.$('> img').remove();
```

Comment:
You must not comments things: or remove them, or keep it, but also why are you removing them?